modules/navier_stokes/src/fvkernels/INSFVMixingLengthReynoldsStress.C

```diff
@@ -37,18 +37,16 @@ INSFVMixingLengthReynoldsStress::INSFVMixingLengthReynoldsStress(const InputPara
   : FVFluxKernel(params),
     _dim(_subproblem.mesh().dimension()),
     _axis_index(getParam<MooseEnum>("momentum_component")),
-    _u_var(dynamic_cast<const INSFVVelocityVariable *>(
-        &_subproblem.getVariable(_tid, params.get<std::vector<VariableName>>("u").front()))),
+    _u_var(dynamic_cast<const INSFVVelocityVariable *>(getFieldVar("u", 0))),
     _v_var(params.isParamValid("v")
-               ? dynamic_cast<const INSFVVelocityVariable *>(&_subproblem.getVariable(
-                     _tid, params.get<std::vector<VariableName>>("v").front()))
+               ? dynamic_cast<const INSFVVelocityVariable *>(getFieldVar("v", 0))
                : nullptr),
     _w_var(params.isParamValid("w")
-               ? dynamic_cast<const INSFVVelocityVariable *>(&_subproblem.getVariable(
-                     _tid, params.get<std::vector<VariableName>>("w").front()))
+               ? dynamic_cast<const INSFVVelocityVariable *>(getFieldVar("w", 0))
                : nullptr),
     _rho(getParam<Real>("rho")),
-    _mixing_len(coupledValue("mixing_length"))
+    _mixing_len(coupledValue("mixing_length")),
```

Comment:
just a note for the future that this isnt using AD. So if the mixing length will not propagate all the contributions to the jacobian when it depends on local variables. It does not right now in our models